hw/ip/otp_ctrl/data/otp_ctrl_img_rma.hjson

```diff
@@ -42,6 +42,12 @@
                     // sw/device/lib/base/hardened.h.
                     value: "0x739",
                 },
+                {
+                    name: "CREATOR_SW_CFG_ROM_EXEC_EN",
```

Comment:
dont' we need to set this to `a` for the other hjsons..? otherwise wouldn't mask rom never run? 

also, should we add this to `test rom`? what do you think?

Reply:
+1 for adding this to the DEV otp config.  It may be nice to add to the test_rom as well to enable debugging with the test_rom init sequence.